automation-english/active_user_input/working-groups-config.yml

```diff
@@ -3,11 +3,18 @@
 # It serves as the only input for the automation process
 # For documentation, refer to the README in this directory
 
+front-matter:
+- https://raw.githubusercontent.com/chaoss/website/master/release/contributors.md
+
+
+end-matter:
+- https://raw.githubusercontent.com/chaoss/website/master/release/release-notes.md
+- https://raw.githubusercontent.com/chaoss/website/master/LICENSE
 
 # WG-COMMON
 
 wg-common:
-  include-wg: True
+  include-wg: False
```

Comment:
We can keep it to `True` I guess

Reply:
The Common WG has a metric `contributors.md` that clashes with the page on the website- [contributors.md](https://github.com/chaoss/website/blob/master/release/contributors.md)

We can keep the flag to `False` till the website gets updated.

Reply:
Makes sense, I'll resolve this and merge
Thank you @yash2002109